src/main/java/com/cosc436002fitzarr/brm/repositories/BuildingRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.cosc436002fitzarr.brm.repositories;
+
+import com.cosc436002fitzarr.brm.models.building.Building;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface BuildingRepository extends MongoRepository<Building, String> {
```

Comment:
What type of custom queries do you think we could use with buildings? Probably not the most important thing right now but eventually we'd probably want to add some custom queries here.